### PR TITLE
SSS-72 organization stu3 support

### PR DESCRIPTION
**Pull request by jnguy-va**
Ready to go~
https://vasdvp.atlassian.net/browse/SSS-72

Support Stu3 for Organization.

### CODE REVIEW

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/organization/Stu3OrganizationController.java [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    if (delimiterIndex <= -1) {
+      throw new ResourceExceptions.BadSearchParameter("Cannot parse NPI for " + systemAndCode);
+    }
+    String system = systemAndCode.substring(0, delimiterIndex);
+    if (!system.equalsIgnoreCase("http://hl7.org/fhir/sid/us-npi")) {
+      throw new ResourceExceptions.BadSearchParameter(
+          String.format("System %s is not supported", system));
+    }
+    String npi = systemAndCode.substring(delimiterIndex + 1);
+    Page<OrganizationEntity> entitiesPage = repository.findByNpi(npi, page(page, count));  //AHHHHHHHHHHHHHHHHHHHHHHHHHHH
```
Comment (jhulbert-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/organization/Stu3OrganizationController.java]: Loud Noises?

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/organization/DatamartOrganization.java [hunk 2]
```diff
@@ -114,6 +114,8 @@
   @NoArgsConstructor(access = AccessLevel.PRIVATE)
   @AllArgsConstructor(access = AccessLevel.PRIVATE)
   public static final class Address {
+    private String text;
```
Comment (jblefkove-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/organization/DatamartOrganization.java]: image

Comment (jblefkove-va) [reply data-query/src/main/java/gov/va/api/health/dataquery/service/controller/organization/DatamartOrganization.java]: I don't see this used anywhere, I am going to remove it

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/organization/Stu3OrganizationTransformer.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            address.postalCode())) {
+      return null;
+    }
+    return asList(
+        Organization.OrganizationAddress.builder()
+            .text(
+                trimToNull(
+                    trimToEmpty(address.line1())
+                        + " "
+                        + trimToEmpty(address.line2())
```
Comment (bschofield-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/organization/Stu3OrganizationTransformer.java]: if line 2 (or anything) is empty, we'll get double spaces

Comment (jblefkove-va) [reply data-query/src/main/java/gov/va/api/health/dataquery/service/controller/organization/Stu3OrganizationTransformer.java]: I addressed this for Location as well

**Pull request merged**